sensor_msgs/Image image_data
---
int8 num_dominos
int8[] num_dots_half1
int8[] num_dots_half2
float32[] x
float32[] y
string[] orientation